plugins/squads/src/components/SquadEntityPage/EntityNotFound/EntityNotFound.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Grid, Button, Typography } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import { BackstageTheme } from '@backstage/theme';
+
+import { Illo } from './Illo';
+
+const useStyles = makeStyles<BackstageTheme>(theme => ({
+  container: {
+    paddingTop: theme.spacing(24),
+    paddingLeft: theme.spacing(8),
+  },
+  title: {
+    paddingBottom: theme.spacing(2),
+  },
+  body: {
+    paddingBottom: theme.spacing(6),
+  },
+}));
+
+export const EntityNotFound = () => {
+  const classes = useStyles();
+
+  return (
+    <Grid container className={classes.container}>
+      <Illo />
+      <Grid item xs={12} sm={6}>
+        <Typography variant="h2" className={classes.title}>
+          Entity was not found
+        </Typography>
+        <Typography variant="body1" className={classes.body}>
+          Want to help us build this? Check out our Getting Started
```

Comment:
This doesn't really make sense for a group

Reply:
This is also copied from the catalog. We could change this to "Group was not found", but I think the good solution would be to have some sort of core components plugin, where this can be imported from and just specify what you want it to say

Reply:
Maybe this is what the core package is for? 😁

Reply:
It's possible that we may move some bits of the catalog plugin into core at some point, but it is also very important that plugins are able to build on each other to add functionality. I think we should figure this out without moving things to core.